Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
+
+  bottle do
+    # sha256 "256" => :high_sierra
+    # sha256 "256" => :sierra
+    # sha256 "256" => :el_capitan
+  end
+
+  # option "with-debug", "Install debugging version with advanced logs"
+
+  depends_on "cmake" => :build
+  depends_on "libiodbc" => :build
+  depends_on "libtool" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl" => :build
+
+  needs :cxx14
+
+  def install
+    args = std_cmake_args
+    # args << "-DCMAKE_BUILD_TYPE=Debug" if build.with? "debug"
+    mkdir "build" do
+      system "cmake", "-G", "Ninja", "..", *args
+      system "ninja"
+      # system "ninja", "install" # installs too many trash
```

Comment:
We prefer to install what upstream considers "standard". Why is it problematic?

Reply:
because we use poco via submodules and poco wants install tons of /include/ and useless for us libs.
I dont know how to completely disable install target for one directory in cmake.